mango/container/mp.py

```diff
@@ -0,0 +1,513 @@
+import asyncio
```

Comment:
I just had to scroll quite a bit to realize what `mp` means in this file name... makese sense but should we maybe still call it 'multi_process'?